atlasaction/action.go

```diff
@@ -585,12 +592,19 @@ func createRunContext(act *githubactions.Action) (*atlasexec.RunContext, error)
 	if branch == "" {
 		branch = ghContext.RefName
 	}
+	var a Actor
+	if err := envconfig.Process(ctx, &a); err != nil {
```

Comment:
I have to do the parsing manually, because the GO SDK doesn't support the `ACTOR_ID` var:
<img width="924" alt="image" src="https://github.com/ariga/atlas-action/assets/63970571/3189d91e-b166-4159-81ba-33abbd40529b">